### PR TITLE
Aug 2019 updates

### PR DESCRIPTION
**Pull request by jtuliani**
August 2019 updates are ready for SME review. Summary of changes:

- Major updates to the HOL to use Azure Migrate v2 instead of Azure Site Recovery. Also now uses DMS instead of DMA for DB schema migration.

- Minor updates to WDS similarly to use Azure Migrate v2 instead of ASR.

### CODE REVIEW

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 1]
```diff
@@ -512,107 +552,102 @@ In this task you will create a new Azure SQL database to migrate the on-premises
...
+#### Task summary <!-- omit in toc -->
 
-6. In the Data Migration Assistant, select the New (+) icon, and then select the **Assessment** project type.
+In this task you created a new Azure Database Migration Service resource.
 
-7. Specify a project name (*e.g.* SmartHotelAssessment), in the **Source server type** text box select **SQL Server**, in the **Target server type** text box, select **Azure SQL Database**, and then select **Create** to create the project.
+### Task 4: Assess the on-premises database
 
-    ![Screenshot showing the new DMA project creation dialog.](Images/Exercise2/new-dma-assessment.png)
+In this task you will install and use Microsoft SQL Server Data Migration Assistant (DMA) to assess the on-premises database.
```
Comment (snehaamicrosoft) [inline Hands-on lab/HOL step-by step - Line-of-business application migration.md]: We can probable add a sentence highlighting the value prop of integration into Azure Migrate, where they can find tools for assessing servers and databases as well as it is integrated with other tools to light up various scenarios.

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 2]
```diff
@@ -621,522 +656,437 @@ In this task you will use Microsoft Data Migration Assistant to migrate the data
...
-8. The script to migrate the schema to Azure SQL Database is shown on the **Script & deploy schema** tab. Review the script and select **Deploy schema** to deploy the schema to Azure SQL Database. Under **Deployment results**, verify that the script executed successfully.
+18. Minimize the remote desktop window and return to the **Azure Migrate - Databases** blade in the Azure portal. Refreshing the page should now show the assessed database.
 
-    ![Screenshot showing the schema migration script in the DMA migration wizard.](Images/Exercise2/deploy-schema.png)
+    ![Screenshot of the 'Azure Migrate - Databases' blade in the Azure portal, showing 1 assessed database.](Images/Exercise2/db-assessed.png)
 
 #### Task summary <!-- omit in toc -->
 
-In this task you used Microsoft Data Migration Assistant to migrate the database schema to Azure SQL. This is a prerequisite to migrating the database contents with the Azure Database Migration Service.
+In this task you used Data Migration Assistant to assess an on-premises database for readiness to migrate to Azure SQL, and uploaded the assessment results to your Azure Migrate project.
```
Comment (snehaamicrosoft) [inline Hands-on lab/HOL step-by step - Line-of-business application migration.md]: We can highlight the value prop that this enables you to centrally track your assessment and migration journey for servers/databases in a single place in Azure Migrate.

**Comment by jtuliani** [comment]
More feedback, from Kunal:

## Point 1
During “Assessment creation” documentation tells about selecting “Yes” for option “Already have windows licenses”. However if we select Yes then “Save option remain disabled on portal; it enables only when you select No. If we select “All VM series” then only it enables the “Save” button. 

Not sure if the documentation is wrong or portal is mis behaving.

## Point 2
Also the dependency agent is not installing if we follow the documentation as is. Refer screenshot about non supported OS version kernel of ubuntu. Do we need to run “apt-get update”? But this will change the OS version also I guess for current VM. Please guide.

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 3]
```diff
@@ -27,57 +27,56 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
...
 
 ## Overview
 
 Before the lab, you will have pre-deployed an on-premises infrastructure hosted in Hyper-V.  This infrastructure is hosting a multi-tier application called 'SmartHotel', using Hyper-V VMs for each of the application tiers.
 
-During the lab, you will migrate this entire application stack to Azure. This will include assessing the on-premises application using Azure Migrate; migrating the database tier to Azure SQL Database using SQL Server Data Migration Assistant (DMA) and the Azure Database Migration Service (DMS); and migrating the web and application tiers using Azure Site Recovery. This includes migration of both Windows and Linux VMs.
+During the lab, you will migrate this entire application stack to Azure. This will include assessing the on-premises application using Azure Migrate; assessing the database migration using SQL Server Data Migration Assistant (DMA); migrating the database using the Azure Database Migration Service (DMS); and migrating the web and application tiers using the Azure Site Recovery migration engine integrated into Azure Migrate. This last step includes migration of both Windows and Linux VMs.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Good fb, will do

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 4]
```diff
@@ -27,57 +27,56 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
...
+  - [Exercise 2: Migrate the Application Database](#Exercise-2-Migrate-the-Application-Database)
+    - [Overview](#Overview-1)
+    - [Task 1: Register the Microsoft.DataMigration resource provider](#Task-1-Register-the-MicrosoftDataMigration-resource-provider)
+    - [Task 2: Create an Azure SQL Database](#Task-2-Create-an-Azure-SQL-Database)
+    - [Task 3: Create the Database Migration Service](#Task-3-Create-the-Database-Migration-Service)
+    - [Task 4: Assess the on-premises database](#Task-4-Assess-the-on-premises-database)
+    - [Task 5: Create a DMS migration project](#Task-5-Create-a-DMS-migration-project)
+    - [Task 6: Migrate the database schema](#Task-6-Migrate-the-database-schema)
+    - [Task 7: Migrate the on-premises data](#Task-7-Migrate-the-on-premises-data)
+  - [Exercise 3: Migrate the application and web tiers using Azure Site Recovery](#Exercise-3-Migrate-the-application-and-web-tiers-using-Azure-Site-Recovery)
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Got it

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 5]
```diff
@@ -88,17 +87,19 @@ The SmartHotel application comprises 4 VMs hosted in Hyper-V:
...
+
+Throughout this lab, you will use Azure Migrate as your primary tool for assessment and migration. In conjunction with Azure Migrate, you will also use a range of other tools, as detailed below.
 
 To assess the Hyper-V environment, you will use Azure Migrate. This includes deploying the Azure Migrate appliance on the Hyper-V host to gather information about the environment. For deeper analysis, the Microsoft Monitoring Agent and Dependency Agent will be installed on the VMs, enabling the Azure Migrate dependency visualization.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Got it

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 6]
```diff
@@ -122,19 +121,19 @@ Refer to the [Before the HOL - Line-of-business application migration](./Before%
 
 Duration: 60 minutes
 
-In this exercise, you will use Azure Migrate to assess the on-premises environment.
+In this exercise, you will use Azure Migrate to assess the on-premises environment. This will include selecting Azure Migrate tools, deploying the Azure Migrate appliance into the on-premises environment, creating a migration assessment, and using the Azure Migrate dependency visualization.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Got it

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 7]
```diff
@@ -621,522 +656,437 @@ In this task you will use Microsoft Data Migration Assistant to migrate the data
...
+In subsequent tasks, you will use this project to migrate both the database schema and the data itself from the on-premises SQL Server database to the Azure SQL Database.
 
-    ![Screenshot showing the DMS 'create' button.](Images/Exercise2/dms-create-1.png)
+1.  Check that the Database Migration Service resource you created in task 3 has completed provisioning. You can check the deployment status from the **Deployments** pane in the **AzureMigrateRG** resource group blade.
 
-   > **Tip**: If the migration service blade will not load, refresh the portal blade in your browser.
+    ![Screenshot showing the AzureMigrateRG - Deployments blade in the Azure portal. The Microsoft.AzureDMS deployment shows status 'Successful'.](Images/Exercise2/dms-deploy.png)
 
-3. In the **Create Migration Service** blade enter the following values and click **Create**.
+2.  Navigate to the Data Migration Service resource blade in **AzureMigrateRG** resource group and click **+ New Migration Project**.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Got it (fixed in 5 places)

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 8]
```diff
@@ -621,522 +656,437 @@ In this task you will use Microsoft Data Migration Assistant to migrate the data
...
+    ![Screenshot of the 'Discover machines' panel from Azure Migrate, showing the 'Finalizing registration...' message.](Images/Exercise3/discover-6.png)
 
-15. After the policy is created click **OK** on the **Replication policy** blade.
+13. Once the registration is complete, close the 'Discover machines' panel.
 
-    ![Screenshot of the ASR replication policy blade.](Images/Exercise3/replication-policy.png)
+    ![Screenshot of the 'Discover machines' panel from Azure Migrate, showing the 'Registration finalized' message.](Images/Exercise3/discover-7.png)
 
-16. Click **OK** again to complete the **Prepare infrastructure** wizard. 
+14. The 'Azure Migrate: Server Replication' panel should now show 5 discovered servers.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Fixed

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 9]
```diff
@@ -621,522 +656,437 @@ In this task you will use Microsoft Data Migration Assistant to migrate the data
...
 
-> **Note:** The Microsoft Azure Virtual Machine Agent (VM Agent) is a secure, lightweight process that manages virtual machine (VM) interaction with the Azure Fabric Controller. The VM Agent has a primary role in enabling and executing Azure virtual machine extensions. VM Extensions enable post-deployment configuration of VM, such as installing and configuring software. VM extensions also enable recovery features such as resetting the administrative password of a VM. Without the Azure VM Agent, VM extensions cannot be used.
->
-> We will install the VM agent on the Hyper-V VMs before they are migrated. You can also install the agent after migration.
+In this exercise you migrated the application database from on-premises to Azure SQL Database. The SQL Server Data Migration Assistant was used for migration assessment, and the Azure Database Migration Service was used for schema migration and data migration.
 
-1.  Using the Azure portal, navigate to the **SmartHotelHost** VM, then click **Connect** and open an RDP session to the VM using the user name **demouser** and password **demo@pass123**.
+## Exercise 3: Migrate the application and web tiers using Azure Site Recovery
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Fixed

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 10]
```diff
@@ -27,57 +27,56 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
...
+    - [Task 3: Configure the Azure Migrate appliance](#Task-3-Configure-the-Azure-Migrate-appliance)
+    - [Task 4: Create a migration assessment](#Task-4-Create-a-migration-assessment)
+    - [Task 5: Configure dependency visualization](#Task-5-Configure-dependency-visualization)
+    - [Task 6: Explore dependency visualization](#Task-6-Explore-dependency-visualization)
+  - [Exercise 2: Migrate the Application Database](#Exercise-2-Migrate-the-Application-Database)
+    - [Overview](#Overview-1)
+    - [Task 1: Register the Microsoft.DataMigration resource provider](#Task-1-Register-the-MicrosoftDataMigration-resource-provider)
+    - [Task 2: Create an Azure SQL Database](#Task-2-Create-an-Azure-SQL-Database)
+    - [Task 3: Create the Database Migration Service](#Task-3-Create-the-Database-Migration-Service)
+    - [Task 4: Assess the on-premises database](#Task-4-Assess-the-on-premises-database)
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md (hunk 1 above)
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md (hunk 2 above)
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 11]
```diff
@@ -27,57 +27,56 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
...
-  - [After the hands-on lab](#after-the-hands-on-lab)
-    - [Task 1: Clean up resources](#task-1-clean-up-resources)
+- [Line-of-business application migration hands-on lab step-by-step](#Line-of-business-application-migration-hands-on-lab-step-by-step)
+  - [Abstract and learning objectives](#Abstract-and-learning-objectives)
+  - [Overview](#Overview)
+  - [Solution architecture](#Solution-architecture)
+  - [Requirements](#Requirements)
+  - [Before the hands-on lab](#Before-the-hands-on-lab)
+  - [Exercise 1: Discover and assess the on-premises environment](#Exercise-1-Discover-and-assess-the-on-premises-environment)
+    - [Task 1: Create the Azure Migrate project](#Task-1-Create-the-Azure-Migrate-project)
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 12]
```diff
@@ -88,17 +87,19 @@ The SmartHotel application comprises 4 VMs hosted in Hyper-V:
...
+![A slide shows the on-premises SmartHotel application architecture. This comprises a SmartHotelHost server running Microsoft Hyper-V. This server hosts 4 VMs: UbuntuWAF, SmartHotelWeb1, SmartHotelWeb2, and SmartHotelSQL1. A series of arrows show how these VMs will be migrated to Azure. The first 3 VMs have an arrow labeled 'Azure Migrate (Azure Site Recovery engine)' pointing to 3 similarly-labeled VMs in Azure. The last VM, SmartHotelSQL1, has an arrow labeled 'Azure Database Migration Service' pointing to an Azure SQL Database. A third arrow labeled 'Azure Migrate' and 'Data Migration Assistant (DMA)' points from all 4 on-premises VMs to an Azure Migrate dashboard showing migration readiness.](Images/overview.png)
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done (image also updated)

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 13]
```diff
@@ -88,17 +87,19 @@ The SmartHotel application comprises 4 VMs hosted in Hyper-V:
...
+The SQL Server database will be assessed by installing the Microsoft SQL Server Data Migration Assistant (DMA) on the Hyper-V host, and using it to gather information about the database. Schema migration and data migration will then be completed using the Azure Database Migration Service (DMS).
 
-The application, web, and web proxy tiers will be migrated to Azure VMs using Azure Site Recovery (ASR). This includes deploying the ASR collector appliance into the Hyper-V environment, building the Azure environment, replicating data to Azure, and performing a failover to migrate the application to Azure.
+The application, web, and web proxy tiers will be migrated to Azure VMs using the Azure Site Recovery (ASR) migration engine integrated into Azure Migrate. You will walk through the steps of building the Azure environment, replicating data to Azure, customizing VM settings, and performing a failover to migrate the application to Azure.
```
[the remark this answers is not in the document]
Comment (jtuliani) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Done

**Comment by jtuliani** [comment]
Regarding above points from Kunal:
#1 is by design. Azure Portal only enables save/discard if a change is made
#2 is 'no-repro'. Works for me. Please try again (perhaps one of the steps was missed?)

**Pull request merged**